ui/selectors/nonce-sorted-transactions-selector.test.js

```diff
@@ -292,7 +313,7 @@ describe('nonceSortedTransactionsSelector', () => {
       {
         nonce: '0x0',
         transactions: txList,
```

Comment:
Why is the result of this test different than the one added above? The inputs look the same to me, so there is something I am not seeing

Reply:
The one above is a legitimate retry of a failed off chain transaction.

this one is a failed off chain followed by a simple send completely new transaction.

Reply:
oh, okay, thanks